orb/core/metamodel.py

```diff
@@ -152,6 +152,11 @@ def store_property(key, value):
             for key, value in attrs.items():
                 if store_property(key, value):
                     attrs.pop(key)
+                elif hasattr(value, '__orb__'):
```

Comment:
`orb.virtual` will register an `__orb__` property to a method that will be understandable to the schema that is generated.